### PR TITLE
Check if the tap device is connection state is as expected

### PR DESCRIPTION
**Pull request by yalzhang**
Check if the tap device is connected to the bridge, and compare
with the expected state.

Signed-off-by: Yalan Zhang <yalzhang@redhat.com>

### CODE REVIEW

File: virttest/utils_libvirt/libvirt_network.py
Comment (chunfuwen) [inline virttest/utils_libvirt/libvirt_network.py:203]: Maybe add else statement: logging.debug('as expected, the tap device is not attached to bridge ')

Comment (yalzhang) [reply virttest/utils_libvirt/libvirt_network.py:203]: Updated, Thank you!

**Review by chunfuwen — APPROVED** [review]
LGTM

**Review by chloerh — APPROVED** [review]
Waiting for another approval to be merged. @Yingshun Would you like to help review this one? Thanks.

**Review by Yingshun — APPROVED** [review]

**Comment by chloerh** [comment]
@Yingshun Thanks for the review. Let's merge this.

**Pull request merged**
